src/Views/Form/Templates/Sequoia/assets/js/form.js

```diff
@@ -184,8 +184,7 @@
 				window.give_global_vars.purchase_loading = '';
 
 				const testNotice = $( '#give_error_test_mode' );
-				$( testNotice ).clone().prependTo( '.give-section.payment' );
-				$( testNotice ).remove();
+				moveErrorNotice( testNotice );
```

Comment:
@henryholtgeerts one question,
do we have to handle this manually?
can the observer `window.MutationObserver` which you already setup take care of this?

Reply:
@ravinderk Because the test notice is displayed on page load, it has to be handled manually. The mutation observer only interacts with nodes that are added to the DOM, so it isn't able to pick up on the test mode notice.

Reply:
@henryholtgeerts thank you for information.